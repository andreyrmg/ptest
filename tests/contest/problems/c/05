5
gridnkz.ipsigho?e,,aohmnm,mglu#
