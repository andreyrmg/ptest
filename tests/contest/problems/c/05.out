555996644487331166888999665556880005111#11133388677787771117775557700